src/wymeditor/editor/image-handler.js

```diff
@@ -0,0 +1,681 @@
+/* jshint maxlen:100 */
+"use strict";
+
+/*
+ * # The Image Handler
+ *
+ * Give it an editor instance and it will make
+ * most of your image resizing dreams come true.
+ *
+ * ## IE8 Shenanigans
+ *
+ * When IE8 is not longer supported,
+ * `rem` could be used for more accurate UI element dimensions
+ *
+ * ## IE9 Shenanigans
+ *
+ * Dragging and dropping of images is disabled.
+ * See the `_isImgDragDropAllowed` function.
+ *
+ * ## IE8-11 Shenanigans
+ *
+ * SVG images are not scaled.
+ * They are cropped. That's right.
+ * And applying style to them does not help,
+ * as well. Ideas are welcome.
+ *
+ * ## General Shenanigans (http://i.imgur.com/wbQ6U5C.jpg)
+ *
+ * This module is covered by only a few basic tests
+ * so any change must be meticulously manually tested
+ * in all the supported browsers
+ * by psychologically stable individuals.
+ *
+ * Dragging and dropping of images
+ * might produce undesired results on drop.
+ * Uncharted territory.
+ *
+ * In event handlers of events that
+ * are not expected to perform useful default actions
+ * `return false` is used to prevent any bad feelings
+ * towards unexpected browser behavior.
+ */
+
+// the image handler class.
+WYMeditor.ImageHandler = function (wym) {
+    var ih = this;
+    ih._wym = wym;
+
+    ih._$resizeHandle = ih._createResizeHandle();
+
+    ih._$currentImageMarker = null;
+
+    // references the image that
+    // has the resize handle placed on it
+    ih._$currentImg = null;
+
+    // flags whether a resize operation is
+    // occurring at this moment
+    ih._resizingNow = false;
+
+    ih._imgDragDropAllowed = WYMeditor.ImageHandler._isImgDragDropAllowed();
+
+    ih._addEventListeners();
+
+    return ih;
+};
+
+WYMeditor.ImageHandler._isImgDragDropAllowed = function () {
+    var browser = jQuery.browser;
+    if (browser.msie) {
+        if (browser.versionNumber === 9) {
+            // dragging and dropping seems to not consistently work.
+            // the image would only some times get picked up by the mouse drag attempt.
+            // to prevent confusion
+            return false;
```

Comment:
Yes. There's something going on under the hood which I didn't figure out.

I have just tried the same trick as with IE11 (selecting the image synchronously `onmousedown`. Didn't seem to help. I'm going to leave it like this.